src/power_grid_model/core/error_handling.py

```diff
@@ -76,6 +77,7 @@
 _INVALID_SHORT_CIRCUIT_PHASE_OR_TYPE_RE = re.compile(r"short circuit type")  # multiple different flavors
 _POWER_GRID_DATASET_ERROR_RE = re.compile(r"Dataset error: ")  # multiple different flavors
 _POWER_GRID_UNREACHABLE_HIT_RE = re.compile(r"Unreachable code hit when executing ")  # multiple different flavors
+_POWER_GRID_SEARCH_OPT_INCMPT_RE = re.compile(r"Binary search method is incompatible with {any} strategy")
```

Comment:
```suggestion
_POWER_GRID_SEARCH_OPT_INCMPT_RE = re.compile(r"Binary search method is incompatible with (.+) #(-?\d+)\n")
```
please double-check that this works with all strategy types

Reply:
This is the 4xchecked result. It's by design only not compatible with `any`

Reply:
and confirmed with tests

Reply:
great :+1: